dev-tools/validation/check-authz-route-matrix.py

```
@@ -114,6 +149,314 @@ def validate_auth_path_vocabulary(document: dict[str, Any], errors: list[str]) -
+        if status is not None and status not in ROUTE_STATUS_VALUES:
+            errors.append(
+                f"routes[{index}] route_status must be one of {sorted(ROUTE_STATUS_VALUES)}"
+            )
+        implementation_status = route.get("implementation_status")
+        if isinstance(implementation_status, dict) and "target_only" in implementation_status:
+            errors.append(
+                f"routes[{index}] must use route_status instead of implementation_status.target_only"
+            )
+
+
+def validate_receiver_predicates(
```

Ruff (0.15.21)

[warning] Too many branches (22 > 12)

(PLR0912)